README.rst

```diff
@@ -65,6 +65,47 @@ The latest code can be installed directly from the Git repository:
 
     $ pip install -e git+https://github.com/SwissDataScienceCenter/renku-python.git#egg=renku
 
+Use following installation steps based on your operating system and preferences
+if you would like to work only with the command line interface.
```

Comment:
Why "only"? As opposed to develop renku?

Reply:
Just wanted to make sure they don't expect the whole platform to be installed.